tests/test_special_euclidean.py

```diff
@@ -140,23 +146,61 @@ def setUp(self):
         self.angles_close_to_pi = self.angles_close_to_pi_all
         if geomstats.tests.tf_backend():
             self.angles_close_to_pi = ['with_angle_close_pi_low']
+        else:
+            self.elements_matrices_all = elements_matrices_all
+            self.elements_matrices = elements_matrices
+
         self.n_samples = 3
 
     def test_random_and_belongs(self):
-        """
+        """Checks random_uniform and belongs
+
         Test that the random uniform method samples
         on the special euclidean group.
         """
         base_point = self.group.random_uniform()
         result = self.group.belongs(base_point)
-        expected = gs.array([[True]])
+        expected = gs.array([True])
+        print(result, expected)
+        self.assertAllClose(result, expected)
+
+    @geomstats.tests.np_only
+    def test_random_and_belongs_matrix_form(self):
+        """
+        Test that the random uniform method samples
+        on the special euclidean group.
+        """
+        local_point_type = 'matrix'
+        base_point_1 = self.group.random_uniform(1, local_point_type)
+        base_point_2 = gs.copy(base_point_1)
+        base_point_3 = gs.copy(base_point_1)
+        # Violates SE(n) structure on the last line
```

Comment:
However, comments are usually fine in unit tests, where we are a bit less strict on the coding style :)